src/ios/FirebasePlugin.m

```diff
@@ -305,7 +305,14 @@ - (void)sendToken:(NSString *)token {
 - (void)logEvent:(CDVInvokedUrlCommand *)command {
     [self.commandDelegate runInBackground:^{
         NSString* name = [command.arguments objectAtIndex:0];
-        NSDictionary* parameters = [command.arguments objectAtIndex:1];
```

Comment:
Also here, I made changes because logEvent() for iOS wasn't working correctly.
https://github.com/RADAR-base/cordova-plugin-firebase/commit/767970211c1c6ed7bf149242936e8c08ec35f8e7#diff-0a7bc366a191d0cde3bbd4e85b5640b4